gravitee-apim-console-webui/src/management/application/creation-ng/components/application-creation-form.component.ts

```diff
@@ -133,4 +140,11 @@ export class ApplicationCreationFormComponent implements OnInit {
       share(),
     );
   }
+
+  private initializeGroups() {
```

Comment:
Why do we need a behaviour subject here ? I mean in the file and in this method. Shouldn't we do it without the behaviour subject ?